### ISSUE TITLE
Adding the gcalendar plugin leads to errors running tsc:full

### ISSUE THREAD

**Issue by iain-b** [backstage/backstage#10508]
Firstly, I really, really like the gcalendar home page card! I had a minor issue integrating it though.

TL;DR I needed to install some type packages.

## Expected Behavior

The installation docs cover installing any required type packages (or I do not get type errors when I install the plugin).

## Actual Behavior

```
yarn tsc:full                       
yarn run v1.22.1
$ backstage-cli clean && tsc --skipLibCheck false --incremental false
node_modules/@backstage/plugin-gcalendar/dist/index.d.ts:2:23 - error TS2688: Cannot find type definition file for '@maxim_mazurok/gapi.client.calendar'.

2 /// <reference types="@maxim_mazurok/gapi.client.calendar" />
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@backstage/plugin-gcalendar/dist/index.d.ts:11:26 - error TS2503: Cannot find namespace 'gapi'.

11 declare type GCalendar = gapi.client.calendar.CalendarListEntry;
                            ~~~~

node_modules/@backstage/plugin-gcalendar/dist/index.d.ts:12:30 - error TS2503: Cannot find namespace 'gapi'.

12 declare type EventAttendee = gapi.client.calendar.EventAttendee;
                                ~~~~

node_modules/@backstage/plugin-gcalendar/dist/index.d.ts:13:31 - error TS2503: Cannot find namespace 'gapi'.

13 declare type GCalendarEvent = gapi.client.calendar.Event & Pick<GCalendar, 'backgroundColor' | 'primary'> & Pick<EventAttendee, 'responseStatus'> & {
```

## Steps to Reproduce

1. Install the plugin (https://github.com/backstage/backstage/tree/master/plugins/gcalendar)
2. `yarn tsc:full`

## Context

Installing the following type packages resolved the issue:

```
    "@types/gapi": "^0.0.41",
    "@types/gapi.auth2": "^0.0.56",
    "@types/gapi.client.calendar": "^3.0.10",
```

I'm not sure if there's a way to get this to work out-of-the-box.

## Your Environment

- Browser Information: 

- Output of `yarn backstage-cli info`: 

```text

```

**Comment by alexrybch** [comment]
Thanks for reporting! Should be fixed with #10531 PR

**Comment by benjdlambert** [comment]
Closing as this should fixed with the above PR.
